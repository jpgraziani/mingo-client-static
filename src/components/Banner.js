import React from 'react';

const Banner = ({children, title}) => {
  return (
    <section className='banner'>
      <h2>{title}</h2>
      {children}
    </section>
  )
}

export default Banner;