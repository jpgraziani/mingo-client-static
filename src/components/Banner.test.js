import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Banner from './Banner'

describe(`Banner component`, () => {
  it('renders a Banner by default', () => {
    const wrapper = shallow(<Banner />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})