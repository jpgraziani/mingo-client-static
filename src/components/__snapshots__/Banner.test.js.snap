// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner component renders a Banner by default 1`] = `
<section
  className="banner"
>
  <h2 />
</section>
`;
